include/libdnf/rpm/solv_sack.hpp

```diff
@@ -62,7 +62,7 @@ class PackageSet;
 struct PackageId {
 public:
     PackageId() = default;
-    explicit PackageId(int id);
+    PackageId(int id);
```

Comment:
May I ask you why you removed explicit? For type control it is better when int (Id) is not automatically converted into PackageId.

Reply:
I have reverted this. Thanks for spotting it.